src/JoomlaBrowser.php

```diff
@@ -899,4 +905,18 @@ public function disableStatistics()
         $I->waitForElement(['link' => 'Never'], 60);
         $I->click(['link' => 'Never']);
     }
+
+	/**
+	 * Method is to set Wait for page title
+	 *
+	 * @param   string   $title    Page Title text
+	 * @param   integer  $waiting  Waiting time
+	 *
+	 * @return  void
+	 */
+	public function waitForPageTitle($title, $waiting = 60)
+	{
+		$I = $this;
+		$I->waitForText($title, $waiting, ['css' => '.page-title']);
```

Comment:
instead of ['css' => '.page-title'] the class locator will perform faster. Try: ['class' => 'page-title']